src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/InputRequireThisValidateOnlyOverlappingFalseLeaves.java

```diff
@@ -0,0 +1,39 @@
+/*
+RequireThis
+checkFields = (default)true
+checkMethods = (default)true
+validateOnlyOverlapping = false
+
+
+*/
+// non-compiled with javac: Compilable with Java14
```

Comment:
Try removing space

Reply:
According to the code, it only cares about the `:` and after.
https://github.com/checkstyle/checkstyle/blob/970256be278713ec84660a432767595c930a8607/.ci/validation.sh#L512

Reply:
I take it back. It looks like it may be the reason Java 11 is picking it up.
https://github.com/checkstyle/checkstyle/blob/970256be278713ec84660a432767595c930a8607/.ci/validation.sh#L502